.github/jest-unit.yml

```diff
@@ -0,0 +1,28 @@
+name: Jest Unit Tests
+
+on:
+  pull_request:
```

Comment:
Only trigger on pull requests to the dev branch - consistent with how the cypress yml is set up